app/controllers/application_controller.rb

```diff
@@ -1,3 +1,11 @@
 class ApplicationController < ActionController::Base
   skip_before_action :verify_authenticity_token
+  # before_action :configure_permitted_parameters, if: :devise_controller?
```

Comment:
What's going on here?

Reply:
Oh oops I forgot to remove that. That was something Elyse told us to add when we were troubleshooting the rspec tests but that was actually already handled in our devise configurations and routes. I'll get rid of it before the merge